verticapy/core/vdataframe/_plotting.py

```diff
@@ -239,6 +252,93 @@ def animated(
 
     @save_verticapy_logs
     def bar(
+        self,
+        columns: SQLColumns,
+        method: str = "density",
+        of: Optional[str] = None,
+        max_cardinality: tuple[int, int] = (6, 6),
+        h: tuple[PythonNumber, PythonNumber] = (None, None),
+        bar_type: Literal["auto", "stacked"] = "auto",
+        ax: Optional[Axes] = None,
+        **style_kwargs,
+    ):
+        """
+    Draws the histogram of the input vDataColumns based on an aggregation.
+
+    Parameters
+    ----------
+    columns: SQLColumns
+        List of the vDataColumns names. The list must have one or two elements.
+    method: str, optional
+        The method to use to aggregate the data.
+            count   : Number of elements.
+            density : Percentage of the distribution.
+            mean    : Average of the vDataColumn 'of'.
+            min     : Minimum of the vDataColumn 'of'.
+            max     : Maximum of the vDataColumn 'of'.
+            sum     : Sum of the vDataColumn 'of'.
+            q%      : q Quantile of the vDataColumn 'of' (ex: 50% to get the median).
+        It can also be a cutomized aggregation (ex: AVG(column1) + 5).
+    of: str, optional
+        The vDataColumn to use to compute the aggregation.
+    max_cardinality: tuple, optional
+        Maximum number of distinct elements for vDataColumns 1 and 2 to be used as 
+        categorical (No h will be picked or computed)
+    h: tuple, optional
+        Interval width of the vDataColumns 1 and 2 bars. It is only valid if the 
+        vDataColumns are numerical. Optimized h will be computed if the parameter 
+        is empty or invalid.
+    bar_type: str, optional
+        The BarChart Type.
+            auto    : Regular BarChart based on 1 or 2 vDataColumns.
+            stacked : Stacked BarChart based on 2 vDataColumns.
+    ax: Axes, optional
+        [Only for MATPLOTLIB]
+        The axes to plot on.
+    **style_kwargs
+        Any optional parameter to pass to the Matplotlib functions.
+
+    Returns
+    -------
+    ax
+        Axes
+
+    See Also
+    --------
+    vDataFrame.bar         : Draws the bar chart of the input vDataColumns based on an aggregation.
+    vDataFrame.boxplot     : Draws the Box Plot of the input vDataColumns.
+    vDataFrame.pivot_table : Draws the pivot table of vDataColumns based on an aggregation.
+        """
+        if isinstance(columns, str):
+            columns = [columns]
+        columns, of = self._format_colnames(columns, of, expected_nb_of_cols=[1, 2])
+        if len(columns) == 1:
+            return self[columns[0]].bar(
+                method=method,
+                of=of,
+                max_cardinality=max_cardinality[0],
+                h=h[0],
+                **style_kwargs,
+            )
+        else:
+            vpy_plt, kwargs = self._get_plotting_lib(
```

Comment:
don't you still need vpy_plt and kwargs when the number of columns is 1?

Reply:
No, we call the vDataColumn method.